third_party/2and3/requests/sessions.pyi

```diff
@@ -59,17 +59,17 @@ _Data = Union[None, bytes, MutableMapping[Text, Text], IO]
 
 class Session(SessionRedirectMixin):
     __attrs__ = ...  # type: Any
-    headers = ...  # type: Optional[MutableMapping[Text, Text]]
+    headers = ...  # type: MutableMapping[Text, Text]
     auth = ...  # type: Union[None, Tuple[Text, Text], Callable[[Request], Request]]
-    proxies = ...  # type: Optional[MutableMapping[Text, Text]]
-    hooks = ...  # type: Optional[MutableMapping[Text, Callable[[Request], Any]]]
-    params = ...  # type: Union[None, bytes, MutableMapping[Text, Text]]
+    proxies = ...  # type: MutableMapping[Text, Text]
+    hooks = ...  # type: MutableMapping[Text, Callable[[Request], Any]]
+    params = ...  # type: Union[bytes, MutableMapping[Text, Text]]
```

Comment:
Where does the `bytes` option come from? That's not in `__init__`.

Reply:
I didn't notice that, I was operating off of the output of mypy's `--strict-undefined`. Based on [the docs](http://docs.python-requests.org/en/master/api/#requests.Session.params) it should always be a `MutableMapping`